WooCommerce/WooCommerceTests/ViewRelated/Shipping Label/WooShipping Create Shipping Labels/WooShippingAddPackageViewModelTests.swift

```diff
@@ -79,7 +79,7 @@ final class WooShippingAddPackageViewModelTests: XCTestCase {
                                                                      predefinedPackages: [predefinedPackage])]
                 let allPredefinedOptions = [WooShippingCarrierPredefinedOptions(carrierID: "usps",
                                                                                 predefinedOptions: predefinedOptions)]
-                completion(.success(WooShippingPackagesResponse(storeOptions: ShippingLabelStoreOptions.fake(),
+                completion(.success(WooShippingPackagesResponse(siteID: siteID,
```

Comment:
@bozidarsevo there were compiler errors when building these tests on the trunk. I suppose some related changes got integrated at different times. I fixed it this way and tests succeeded.

Reply:
Weird how https://github.com/woocommerce/woocommerce-ios/pull/14687/ did not fail with compiling :/

Reply:
@rachelmcr made a separate PR with only this fix https://github.com/woocommerce/woocommerce-ios/pull/14722#pullrequestreview-2511633328